string command
---
int32 response
